docs/sphinx/user_guide/tutorial.rst

```diff
@@ -183,6 +215,8 @@ with respect to RAJA usage. We describe them here.
      RAJA::forall<RAJA::cuda_exec>(range, [=] __device__ (int i) {
        // access entries of bounds.array
      } );
+
+   This may be resolved in a future release of the nvcc compiler.
```

Comment:
Would it be worth noting that 10.1 has that fix? IIRC we verified that at some point.

Reply:
@trws good point. I adjusted the tutorial slides to reflect that, but I missed it here. I will commit a change for this shortly. Thanks.

Reply:
Modification pushed.